config/locales/en/models/feedback.yml

```diff
@@ -0,0 +1,9 @@
+---
+en:
+  activemodel:
+    errors:
+      models:
+        feedback:
+          attributes:
+            rating:
+              invalid: Choose a rating
```

Comment:
```suggestion
              inclusion: Choose a rating
```

Reply:
in line with this [validation change](https://github.com/ministryofjustice/laa-court-data-ui/pull/169/files#r440063616)

Reply:
@jsugarman I have accepted your suggestion and will display the Choose a rating message on inclusion and not invalid.